nilearn/decomposition/tests/test_canica.py

```diff
@@ -1,8 +1,11 @@
 """Test CanICA"""
-import nibabel
+
 import numpy as np
 from numpy.testing import assert_array_almost_equal
-from nose.tools import assert_true
+from nose.tools import assert_true, assert_raises
+
+import nibabel
```

Comment:
No, this is what I do too: one block for standard / system imports, one for scientific package and one for inner modules. I was just wondering why you added nibabel but I can see that you removed it from the top of the file.